### ISSUE TITLE
typescript definition in package.json causes tsd install to misbehave?

### ISSUE THREAD

**Issue by myitcv** [palantir/tslint#606]
We've just updated to `2.5.0-dev3` and _I think_ got bitten by [this change](https://github.com/palantir/tslint/commit/c510503fcdf87efe60bfba0e0b63e422b4a86f66#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)

Our `tsd.d.ts` file does not contain a reference to `typescriptServices.d.ts` and hence the namespace `ts` is unavailable. Correspondingly we see our build fail:

```
node_modules/tslint/lib/tslint.d.ts(222,31): error TS2503: Cannot find namespace 'ts'
```

But then I was confused as to how we were even referencing `tslint.d.ts` in the first place. 

From what I can tell, this move to add the `typescript` block to `package.json` triggers `tsd install` to automatically add a reference into an app's `tsd.d.ts` (I might have this wrong). Can't be removed... because `tsd install` (which is what we obviously run in our CI) reinstates it.

Any thoughts on how we go about fixing this?

I'm not keen on referencing `typescriptServices.d.ts` because we don't need it (right now)

Thanks

**Comment by FalloutPL** [comment]
after dozens of changing versions (node, npm, tsd, tslint, typescript) i finnaly could do this:

node: v0.12.7
npm: 2.14.0
tsd: 0.6.3
tslint: 2.4.3
tsc: 1.4.1 (package)
tsc: 1.5.3 (global)

otherwise i had same error. Horribly headache.

**Comment by myitcv** [comment]
Related https://github.com/DefinitelyTyped/tsd/issues/204

**Comment by ashwinr** [comment]
@adidahiya we should revert https://github.com/palantir/tslint/commit/c510503fcdf87efe60bfba0e0b63e422b4a86f66 until https://github.com/DefinitelyTyped/tsd/issues/204 is fixed.

**Comment by myitcv** [comment]
@ashwinr thanks that's what I was about to suggest...

**Comment by myitcv** [comment]
Thanks @adidahiya - when do you plan to cut another release?

**Comment by adidahiya** [comment]
will cut a release now, stand by

**Comment by myitcv** [comment]
Thanks. Saw 2.4.4. Any chance you could also cut 2.5.0-dev4 (which is what
we're tracking)?

**Comment by adidahiya** [comment]
yep, just published `v2.4.4` and `v2.5.0-dev.4` to npm

**Comment by myitcv** [comment]
@adidahiya thanks very much
